tests/legacy/test_gousto.py

```diff
@@ -1,15 +1,16 @@
 from responses import GET
 
 from recipe_scrapers.gousto import Gousto
-from tests import ScraperTest
+
+from . import ScraperTest
```

Comment:
Maybe contentious: I'm not a fan of relative imports in Python, perhaps with the exception of rare cases where an `__init__` file assembles a bunch of stuff to export from the module it is in.

```suggestion
from tests.legacy import ScraperTest
```